gh-default-branch

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# for debug
+cargo build --release
+target/release/gh-default-branch "$@"
```

Comment:
📝 
When both script and binary are present in the repository, gh extension install will give priority to binary, so there is no problem with placing script for debugging.

- https://github.com/cli/cli/blob/v2.10.1/pkg/cmd/extension/manager.go#L338-L357
- https://github.com/cli/cli/pull/4308
- https://github.com/cli/cli/issues/4194#issuecomment-922015391